content/src/mappings/taxClearanceCertificateIssuingAgencies.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
There's a lot of overlap between this list and the `taskAgency.json` list. I'm wondering if there's something clever we can do to consolidate these at some future point. Maybe a good refactor/tech debt item?

Reply:
We can put in a ticket in for this debt item

Reply:
will make a ticket for this Friday 3/7 and link it here

Reply:
https://dev.azure.com/NJInnovation/Legacy%20Pivotal%20Data/_workitems/edit/13224